pos_default_empty_image/static/src/js/widgets.js

```diff
@@ -20,16 +20,19 @@ odoo.define('pos_default_empty_image.widgets', function (require) {
                 return this._super(product);
             }
             else {
```

Comment:
there's no need to `else`, the funtion is already returned otherwise

Reply:
@chienandalu else is necessary here because the rendering is different `QWeb.render('ProductNoImage'...`

Reply:
no, it's not. This structure:
```javascript
if (one_thing) {
  return one_thing
} else {
  return other
}
```
is the same as:
```javascript
if (one_thing) {
  return one_thing
}
return other
```
Preferring the second one as it's less complex